implementations/index.md

```diff
@@ -166,7 +166,7 @@ and writing of JSON:API documents.
 * [paknahad/jsonapi-bundle](https://github.com/paknahad/jsonapi-bundle) is a Symfony bundle. It is the fastest way to generate API.
 * [swisnl/json-api-server](https://github.com/swisnl/json-api-server) is a Laravel package to get a JSON:API up and running in minutes.
 * [hackerboy/json-api](https://github.com/hackerboydotcom/json-api) is a lightweight library that helps you to implement JSONAPI easily
-* [`drupal/jsonapi`](https://www.drupal.org/project/jsonapi) is a Drupal module that exposes all data managed by Drupal (entities) according to the JSON:API specification. [`jsonapi_extras`](https://www.drupal.org/project/jsonapi_extras) is an optional extra module to change resource type names, field names and more. And [`openapi`](https://www.drupal.org/project/openapi) is another optional module, that is able to generate an OpenAPI/Swagger representation of the API provided by the `jsonapi` module with a ReDoc-powered UI.
+* [drupal](https://www.drupal.org/project/drupal) The Drupal CMS supports exposing all data it manages (entities) according to the JSON:API specification with no additional modules required as of 8.7.0. [`jsonapi_extras`](https://www.drupal.org/project/jsonapi_extras) is an optional extra module to change resource type names, field names and more. And [`openapi`](https://www.drupal.org/project/openapi) is another optional module, that is able to generate an OpenAPI/Swagger representation of the API provided by the `jsonapi` module with a ReDoc-powered UI. And [`jsonapi`](https://www.drupal.org/project/jsonapi) is a Drupal module that allows versions of Drupal 8 prior to 8.7.0 to expose data via JSON:API.
```

Comment:
Thanks @Eli-TW!

I think it makes sense to include JSON:API Extras, but I think it's outside the scope of this listing to talk about specific version of Drupal core or the `openapi` module. Would you mind removing the version-dependent stuff and OpenAPI/Redoc language?

(I know you didn't add it yourself ;) )

Reply:
Done!